django_cloud_deploy/cli/io.py

```diff
@@ -14,10 +14,75 @@
 """Class for managing console I/O."""
 
 import abc
+import contextlib
 import getpass
 import os
 import re
 import sys
+import threading
+import time
+
+from progress import bar
+
+
+class _ProgressBar(object):
+    """A progress bar showing status of a task.
+
+    Output of the progress bar will be like the following:
+        Processing █████████████████∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙ 55%
```

Comment:
You aren't going to include a time estimate? (at least optionally)

Reply:
I added time estimate.